### PR TITLE
DOCSUP-28960: replace performer and user with Toloker in docs

### PR DESCRIPTION
**Pull request by lehasm**

### CODE REVIEW

File: src/client/project/localization.py [hunk 1]
```diff
@@ -46,10 +46,10 @@ class Source(Enum):
 
 
 class LocalizationConfig(BaseTolokaObject):
-    """Translates the part of the project visible to the performers into different languages
+    """Translates the part of the project visible to Tolokers into different languages
 
-    It is used to make it easier for performers from other countries who do not speak the necessary language to
-    understand and perform tasks.
+    It is used to make it easier for Tolokers from other countries who do not speak the necessary language to
+    understand and do tasks.
```
Comment (alexandervnuchkov) [inline src/client/project/localization.py:52]: Please replace `do` with `complete`

File: src/client/skill.py [hunk 2]
```diff
@@ -10,26 +10,26 @@
...
-    """A skill is an assessment of some aspect of a user's responses (a number from 0 to 100)
+    """A skill is an assessment of some aspect of a Toloker's responses (a number from 0 to 100)
 
     Skill is a general grouping entity, for example "image annotation", which is created once and then used.
-    To set the Skill value for a specific performer, use UserSkill.
-    You can set up skill calculation in a quality control rule, or manually set the skill level for a user.
-    You can use skills to select users who perform pool tasks.
+    To set the Skill value for a specific Toloker, use UserSkill.
+    You can set up skill calculation in a quality control rule, or manually set the skill level for a Toloker.
+    You can use skills to select Tolokers which are allowed to do your tasks.
```
Comment (alexandervnuchkov) [inline src/client/skill.py:18]: Please replace `do` with `complete`

File: src/client/user_restriction.py [hunk 3]
```diff
@@ -84,15 +84,15 @@ class Scope(ExtendableStrEnum):
 
 @inherit_docstrings
 class AllProjectsUserRestriction(UserRestriction, spec_value=UserRestriction.ALL_PROJECTS):
-    """Forbid the performer from doing tasks from all your projects
+    """Forbid the Toloker to do tasks from all your projects
```
Comment (alexandervnuchkov) [inline src/client/user_restriction.py:87]: Please replace `do` with `complete`

File: src/client/user_restriction.py [hunk 4]
```diff
@@ -84,15 +84,15 @@ class Scope(ExtendableStrEnum):
...
     """
 
     pass
 
 
 @inherit_docstrings
 class PoolUserRestriction(UserRestriction, spec_value=UserRestriction.POOL):
-    """Forbid the performer from doing tasks from a specific pool
+    """Forbid the Toloker to do tasks from a specific pool
```
Comment (alexandervnuchkov) [inline src/client/user_restriction.py:95]: Please replace `do` with `complete`

File: src/client/user_restriction.py [hunk 5]
```diff
@@ -103,7 +103,7 @@ class PoolUserRestriction(UserRestriction, spec_value=UserRestriction.POOL):
 
 @inherit_docstrings
 class ProjectUserRestriction(UserRestriction, spec_value=UserRestriction.PROJECT):
-    """Forbid the performer from doing tasks from a specific project
+    """Forbid the Toloker to do tasks from a specific project
```
Comment (alexandervnuchkov) [inline src/client/user_restriction.py:106]: Please replace `do` with `complete`

**Review by alexandervnuchkov — CHANGES_REQUESTED** [review]

File: src/client/quality_control.py [hunk 6]
```diff
@@ -14,7 +14,7 @@
 class QualityControl(BaseTolokaObject):
     """Quality control unit settings and pool ID with training tasks
 
-    Quality control lets you get more accurate responses and restrict access to tasks for cheating performers.
+    Quality control lets you get more accurate responses and restrict access to tasks for cheating users.
```
Comment (alexdrydew) [inline src/client/quality_control.py:17]: Why not Tolokers?

**Review by alexdrydew — CHANGES_REQUESTED** [review]

File: src/client/quality_control.py (hunk 6 above)
Comment (lehasm) [reply src/client/quality_control.py:17]: I have seen recommendation not to use _Tolokers_ with words like _fraudulent_ and so on. And _cheating_ has similar sense. I do not insist, anyway.
@alexandervnuchkov, comment, please.

Comment (lehasm) [reply src/client/quality_control.py:17]: Although, I overlooked it and changed collectors.py with _Filtering cheating Tolokers who respond too quickly_

**Review by alexdrydew — APPROVED** [review]

**Pull request merged**